src/test/java/StepDefinition/HotelsHelpEmptyFeedbackErrorSD.java

```diff
@@ -0,0 +1,45 @@
+package StepDefinition;
+
+
+import Helper.Check;
+import Pages.Hotels.HelpPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+
+public class HotelsHelpEmptyFeedbackErrorSD {
+
+    HelpPage hp = new HelpPage();
+
+    @When("^I click the 'Help' button in the hotels webpage$")
+    public void clickHelpButtonInHotelPage() {
+    hp.clickHelpButton();
+    }
+
+    @And("^I select the 'Website feedback' option$")
```

Comment:
As I mentioned earlier, Pls don't use @And at SD files, rather use @When or @Then whichever appropriate.